mozilla-release/mobile/android/app/src/photon/res/layout/two_line_page_row.xml

```diff
@@ -2,84 +2,95 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-
+<!-- Cliqz Start -->
 <merge xmlns:android="http://schemas.android.com/apk/res/android"
        xmlns:gecko="http://schemas.android.com/apk/res-auto"
        xmlns:tools="http://schemas.android.com/tools"
+       android:orientation="vertical"
```

Comment:
this merge block will be deleted and all it's children will added to a parent view.
so the parent view should has android:orientation="vertical"